model/sql/com/chiralbehaviors/CoRE/schema/ruleform/tables/status_code_network_authorization.xml

```diff
@@ -50,7 +50,7 @@
         <addForeignKeyConstraint baseTableSchemaName="ruleform " referencedTableSchemaName="ruleform"
             baseColumnNames="classifier" baseTableName="status_code_network_authorization" constraintName="status_code_network_authorization_classification_fkey"
             deferrable="false" initiallyDeferred="true" onDelete="CASCADE" onUpdate="CASCADE" referencedColumnNames="id"
-            referencedTableName="status_code" />
```

Comment:
constraintName="status_code_network_authorization_classification_fkey"

That is the classification. The one below is the classifier